jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/Build.java

```diff
@@ -38,24 +43,73 @@
 
   @CommandLine.Spec private CommandSpec spec = CommandSpec.create();
 
+  @CommandLine.Mixin
+  @SuppressWarnings("NullAway.Init") // initialized by picocli
+  CommonCliOptions commonCliOptions;
```

Comment:
Can this be `private`? Or `@VisibleForTesting` if required?

Reply:
Oh yes, thanks for pointing it out.